src/components/v5/common/ActionsContent/partials/SocialLinksTable/SocialLinksTable.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import { useFieldArray, useFormContext, useWatch } from 'react-hook-form';
+
+import SocialLinksModal from './partials/SocialLinksModal';
+import { useActionSidebarContext } from '~context/ActionSidebarContext';
+import { useGetTableMenuProps, useSocialLinksTableColumns } from './hooks';
+import { SocialLinksTableModel, SocialLinksTableProps } from './types';
+import { formatText } from '~utils/intl';
+import TableWithMeatballMenu from '~v5/common/TableWithMeatballMenu';
+import Button from '~v5/shared/Button/Button';
+import { useMobile } from '~hooks';
+
+const displayName = 'v5.common.ActionsContent.partials.SocialLinksTable';
+
+const SocialLinksTable: FC<SocialLinksTableProps> = ({ name }) => {
+  const {
+    socialLinksModalToggle: [, { toggleOn: toggleChangeSocialLinksModalOn }],
+  } = useActionSidebarContext();
+  const isMobile = useMobile();
+  const fieldArrayMethods = useFieldArray({ name });
+  const value = useWatch({ name });
+  const getMenuProps = useGetTableMenuProps(fieldArrayMethods, value);
+  const columns = useSocialLinksTableColumns(name);
+  const data: SocialLinksTableModel[] = fieldArrayMethods.fields.map(
+    ({ id }) => ({
+      key: id,
+    }),
+  );
+  const { getFieldState } = useFormContext();
+  const fieldState = getFieldState(name);
+
+  return (
+    <>
+      {!!data.length && (
+        <TableWithMeatballMenu<SocialLinksTableModel>
+          tableClassName={clsx({ '!border-red-400': fieldState.error })}
```

Comment:
border-red-400 -> border-negative-400, we don't have red-400 color in tailwind config

Reply:
fixed